lib/ts/controllers/s-modal.ts

```diff
@@ -0,0 +1,88 @@
+class ModalController extends Stacks.StacksController {
+    static targets = ["modal"];
+
+    // TODO should Stimulus bind this? Needed to keep TS error-free
+    private modalTarget!: HTMLElement;
+
+    private _boundClickFn!: any;
+    private _boundKeypressFn!: any;
+
+    connect () { };
+
+    toggle (e: Event) {
```

Comment:
Could we add `show` and `hide` for clarity? `data-action="s-modal#toggle"` feels weird on close buttons.

Reply:
Added both and changed `toggle` to `hide` on the close buttons to make it more clear